desktop/test/e2e/specs/login.js

```diff
@@ -119,7 +119,6 @@ describe( 'User Can log in', () => {
 
 		if ( electronApp ) {
 			try {
-				await electronApp.context().close();
```

Comment:
This was removed at [suggestion](https://github.com/microsoft/playwright/issues/11920#issuecomment-1032128970) by the Playwright maintainers.